showcase how to change charset (#4457)
platformsh/platformsh-docs: sites/platform/src/create-apps/web/custom-headers.md

web:
  locations:
    "/":
      #...
      rules:
        \.md$:
          headers:
            Content-Type: "text/markdown; charset=UTF-8"
```

This rule sets an explicit content type for files that end in `.md`.  Because specific rules override the general
heading configuration, Markdown files don't get the `X-Frame-Options` header set before.

{{< note theme="info" title="Setting charset" >}}

> By default, [HTTP charset parameter](https://www.w3.org/International/articles/http-charset/index.en) are not sent to the response.

Reviewer: ```suggestion
By default, [HTTP charset parameters](https://www.w3.org/International/articles/http-charset/index.en) are not sent to the response.
```

> If not set, modern browsers detect ISO-8859-1 and it normally defaults to Windows-1252, because Windows-1252 has 32 more international characters.

Reviewer: ```suggestion
If not set, modern browsers will detect ISO-8859-1 and likely default to Windows-1252 as this has 32 more international characters.
```

> To set it to the desired charset you can add ``; charset=UTF-8`` in the `Content-Type` parameters.

Reviewer: ```suggestion
To set the HTTP charset parameters to your desired charset, you can add ``; charset=UTF-8`` in the `Content-Type` parameters.
```

Reviewer: I'm not sure with this last suggestion I've made, but my aim is to just make it clear to the user what they're setting instead of saying 'to set it' if that makes sense? Happy to adjust!

Author: that's perfect. 
Yes, this is just because i was struggling to set it for the llms.txt thing and i wanted to add a special note about it (i didn't check if this note is raised in the search engine when looking for `charset`... to be continued)
